misc_docs/syntax/extension_embed_raw_top_level.mdx

```diff
@@ -0,0 +1,43 @@
+---
+id: "embed-raw-top-level"
+keywords: ["javascript", "embed", "raw"]
+name: "%%raw"
+summary: "This is the `embed top level JavaScript` extension point."
+category: "extensionpoints"
+---
+
+`%%raw` is used to embed top level JavaScript code.
+
+<CodeTab labels={["ReScript", "JS Output"]}>
+
+```res
+%%raw(`
+  const message = "hello";
+
+  function greet(m) {
+    console.log(m)
+  }
+
+  greet(message)
+`)
+```
+
+```js
+const message = "hello";
+
+function greet(m) {
+  console.log(m);
+}
+
+greet(message);
+```
+
+</CodeTab>
+
```

Comment:
~~~suggestion

It's also very useful to do imports with side-effects like this:

```res
%%raw(`import "main.css"`)
```

~~~

Reply:
Great, I've included this change 👍